server/server.go

```diff
@@ -648,6 +648,10 @@ func New(opts *Options) *Server {
 func NewServer(opts *Options) (*Server, error) {
 	setBaselineOptions(opts)
 
+	if serverVersion == _EMPTY_ {
+		serverVersion = VERSION
```

Comment:
seems that syft only requires the `serverVersion` / `Version` being included in the ldflags which is addressed in the goreleaser yaml above. 
I wonder whether we should be doing this here after all? We already have a test `TestVersionMatchesTag` that it is called previous to releasing so the released version should already be a tag.

Reply:
yeah. I can probably simplify it.

Reply:
@wallyqs the only issue is that now, two different versions could be reported if the code doesn't get updated in the tagged release.

Reply:
agree but we have check in place to prevent that so I think we could skip for now